content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -8,17 +8,10 @@ aliases:
   -  /hosting/vertical-vs-horizontal
 ---
 
-{{< callout >}}
-For older versions of the CHT, Medic recommended Kubernetes for all production deployments. This is no longer the recommendation for newer versions of the CHT. Further, multi-node CouchDB is no longer recommended.
-{{< /callout >}}
+To deploy the CHT, you should use the technology you are most familiar and comfortable with. It is possible to deploy 20 CHT instances without using Kubernetes. Conversely, if a hosting organisation uses Kubernetes and intends to deploy a single instance of the CHT, that's also fine.
```

Comment:
I think it's important we keep this advice. Can you weave back in somewhere?

> Further, multi-node CouchDB is no longer recommended.

It's otherwise not clear that we no longer recommend it if we remove all mention of it (though I know there's plenty of mention of it on the k8s docs pages)

Reply:
Fair point; I added language used by @dianabarsan in https://github.com/medic/cht-docs/pull/1891/commits/7d774c18943a85306e4cdf46dad4b00412e8dca3